<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-IP7TE20" failures="1" tests="4" name="PetTest" time="17.356" errors="0" timestamp="2025-06-17T11:17:58 IST">
  <testcase classname="api.test.UserTests" name="testPostUser" time="5.342"/>
  <testcase classname="api.test.UserTests" name="testGetUserByName" time="2.182"/>
  <testcase classname="api.test.UserTests" name="testUpdateUserByName" time="4.388">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.test.UserTests.testUpdateUserByName(UserTests.java:74)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- testUpdateUserByName -->
  <testcase classname="api.test.UserTests" name="testDeleteUserByName" time="3.103"/>
</testsuite> <!-- PetTest -->
